lib/plugins/command_block.js

```diff
@@ -1,101 +1,114 @@
+const assert = require('assert')
+
 const ProtoDef = require('protodef').ProtoDef
 
 module.exports = inject
 
 function inject (bot) {
-  function setCommandBlock (pos, command, trackOutput) {
-    const pluginChannelName = bot.supportFeature('usesAdvCdm') ? 'MC|AdvCdm' : 'MC|AdvCmd'
-
-    const proto = new ProtoDef()
+  function setCommandBlock (pos, command, trackOutput, flags) {
```

Comment:
Can you update the doc with this flags parameter please ?

Reply:
The thing is that I'm not sure which parameters it accepts and what they do
https://wiki.vg/Protocol#Update_Command_Block

Reply:
ah yes sorry I forgot this pr
so basically if you don't know what it does, probably you should expose it in the API
mineflayer should abstract away as much as possible the protocol details, so either remove that flags from the exposed API, either clarify in the doc what values it take and make sure that's stable with versions